index.js

```diff
@@ -102,19 +102,21 @@ function fastifyPostgres (fastify, options, next) {
     if (db[name]) {
       return next(new Error(`fastify-postgres '${name}' is a reserved keyword`))
     } else if (!fastify.pg) {
-      fastify.decorate('pg', {})
+      fastify.decorate('pg', Object.create(null))
```

Comment:
I'm curious, what is the advantage of this approach?

Reply:
When we use `Object.create(null)` we initialize a really empty object that inherit nothing.
Initializing the decorator with `{}` instead is equivalent to `Object.create(Object.prototype)` and we have an object that inherit the `__proto__` property.
But in fact it can be considered to be just a minor optimization.